src/state/airdrop/selectors.ts

```diff
@@ -1,10 +1,31 @@
 import { createSelector } from 'reselect';
 
+import { Wallet } from 'app/consts';
+import { HDSegwitP2SHAirWallet } from 'app/legacy';
 import { ApplicationState } from 'app/state';
 
+import { WalletsState } from '../wallets/reducer';
 import { AirdropState } from './reducer';
 
 const local = (state: ApplicationState): AirdropState => state.airdrop;
+const walletsState = (state: ApplicationState): WalletsState => state.wallets;
 
 export const thankYouSeen = createSelector(local, state => state.thankYouSeen);
 export const thankYouFlowCompleted = createSelector(local, state => state.thankYouFlowCompleted);
+export const airdropReadyWallets = createSelector(walletsState, state => {
+  return state.wallets.length > 0
```

Comment:
return state.wallets.length > 0 not needed filter for an empty array returns empty array

Reply:
done